### DISCUSSION TITLE
B.4.2. Agility & Innovation - EKG/Maturity

### DISCUSSION THREAD

**Discussion by giscus** [EKGF/ekg-maturity#74] [Page Comments]
# B.4.2. Agility & Innovation - EKG/Maturity

Maturity Model for the Enterprise Knowledge Graph (EKG)

https://maturity.ekgf.org/pillar/organization/organizational-culture/agility-and-innovation/

**Comment by Fodenad** [comment]
I've been mulling over the concept of Agile software engineering and how it's transformed the way we approach software development. The Agile methodology, with its emphasis on adaptability, collaboration, and customer satisfaction, has brought a breath of fresh air to the industry.

What I appreciate most about Agile is its flexibility. It allows teams to adapt to changes and evolve their products based on customer feedback, market trends, and lessons learned during the development process.

When looking for a web and software development company, it's important to consider their approach to Agile. A company that truly embraces Agile will be able to deliver quality, user-focused products in a timely manner. I recently found a company called Scimus ([https://thescimus.com](https://thescimus.com)), and their commitment to Agile methods seems quite comprehensive, which definitely stands out.

What are your experiences with Agile software engineering? How do you feel it impacts the development process and final product? Can't wait to read your insights!
